packages/mdc-button/_mixins.scss

```diff
@@ -95,7 +95,7 @@
   @include mdc-ripple-radius-bounded;
   @include mdc-button-horizontal-padding($mdc-button-horizontal-padding);
 
-  display: inline-block;
+  display: inline-flex;
```

Comment:
I think you might want to change 
text-align: center; to justify-content: center;